app/javascript/components/base/base-dropdown.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <div>
+    <button
+      class="focus:outline-none"
+      @click="openDropdown"
+    >
+      <div class="flex justify-between items-center min-w-max w-16">
+        <div> {{ selectedElement }} </div>
+        <div> 
+          <img
+            class="w-4 h-4"
+            src="../../../assets/images/chevron-down-svg.svg"
+          >
+        </div> 
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Trailing spaces not allowed   no-trailing-spaces